### PR TITLE
Update README.md docs #15

### LINKED ISSUE #14: Update the README.MD. Justification: As we first started the building process, we were confused by the installation guide as it was not organized. As the README.md is often the first thing potential users or contributors see, a well-maintained README.md can attract interest and help promote the project effectively. So, we felt that a well-written README file should be one of our main objectives..

### LINKED ISSUE #15: Reorganize and proofread README.MD

### PR DESCRIPTION
**Pull request by Jerry4753**
# Pull Request to Resolve User Story #14 
### What?
Modifications to the .md files in multiple directories

---

### Why?
As we first started the building process, we were confused by the installation guide as it was not organized. As the README.md is often the first thing potential users or contributors see, a well-maintained README.md can attract interest and help promote the project effectively. 

---

### How?
- Reorganized various sections within the docs
- Gave numberings and bullet points to promote the chronological sequences needed during the installation process
- Proofread the documents for grammatical errors

---

#### Files Modified:
- ./README.MD
- ./docs/1. How-to-Use-JPlag.md
- ./docs/3. Contributing-to-JPlag.md
- ./docs/5. End-to-end-testing.md

#### This resolves User story #14 by:
- Resolves Issue #15 

---

### TL;DR:
All together, this pull request implements user story #14 . This adds additional clarifications as well as re-structure the installation process to help potential users/contributors understand how to get JPlag up and running.

### CODE REVIEW

**Review by asturnley — APPROVED** [review]
This is very helpful information that we had to work through when working to run this codebase. I think this will be very helpful future users and contributors!

**Review by JavaNoTea — APPROVED** [review]
Adds great and very useful information! I also like the formatting changes, makes the readme a lot more readable!
